packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -275,23 +294,79 @@ function FontCollection( { slug } ) {
 			) }
 
 			{ ! renderConfirmDialog && ! selectedFont && (
-				<FontsGrid>
-					{ fonts.map( ( font ) => (
-						<FontCard
-							key={ font.font_family_settings.slug }
-							font={ font.font_family_settings }
-							onClick={ () => {
-								setSelectedFont( font.font_family_settings );
-							} }
-						/>
-					) ) }
-				</FontsGrid>
+				<VStack spacing={ 0 }>
```

Comment:
Isn't this `VStack` unnecessary?

Reply:
It was indeed unnecessary.  Removed.  Thank you.